rsocket/internal/InMemResumeManager.h

```diff
@@ -55,14 +56,53 @@ class InMemResumeManager : public ResumeManager {
     return impliedPosition_;
   }
 
-  void onStreamOpen(StreamId streamId, FrameType frameType) override;
-
   void onStreamClosed(StreamId streamId) override;
 
+  std::set<StreamId> getRequesterRequestStreamIds() override;
+
+  uint32_t getPublisherAllowance(StreamId streamId) override {
+    return publisherAllowance_[streamId];
```

Comment:
This will insert an entry into the map if it isn't there.  Is that desirable?  I figure it'd be better to do something like `auto it = publisherAllowance_.find(streamId); return it != publisherAllowance_.end() ? it->second : 0;`